po/pt_BR.po

```diff
@@ -27,15 +27,15 @@ msgstr "Gerenciador de Extensões"
 #: data/com.mattjakeman.ExtensionManager.appdata.xml.in:7
 #: data/com.mattjakeman.ExtensionManager.appdata.xml.in:11
 msgid "A utility for browsing and installing GNOME Shell Extensions."
-msgstr "Um utilitário para buscar e instalar as extensões do Gnome Shell"
+msgstr "Um utilitário para buscar e instalar as extensões do Gnome Shell."
```

Comment:
This isn't something that your PR has introduced, but we should fix this anyway. GNOME should be fully capitalised rather than written as a word.

```suggestion
msgstr "Um utilitário para buscar e instalar as extensões do GNOME Shell."
```